Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_BuildIndex.1.txt

```diff
@@ -1219,208 +1219,6 @@
               </div>
             </li>
           </ul>
-          <hr/>
```

Comment:
Could you refresh the corresponding image snapshot please? :)

Reply:
I can't say why for sure, but I just regenerated the image snapshots and nothing changed. A missing `hr` *should* change them. Investigating.

Reply:
Figured it out. The length of the snapshot test isn't big enough to capture this low down in the matrix. It's fine to leave it like this, the HTML lines snapshot will capture any significant changes, and the styling is just repeated.

<img width="1200" alt="test_BuildIndex desktop" src="https://user-images.githubusercontent.com/5180/118641694-5f9be700-b7d2-11eb-880d-14f71caa009a.png">